regex-syntax/src/hir/mod.rs

```diff
@@ -2589,6 +2589,8 @@ impl Properties {
             literal: false,
             alternation_literal: false,
             contains_lookaround_expr: true,
+            look_set_prefix: LookSet::empty(),
+            look_set_suffix: LookSet::empty(),
```

Comment:
Maybe this should be explained with a comment?
Also, did you test/check whether this has an effect on whether or not the assertions work correctly inside a look-behind?

Reply:
@shilangyu I added a comment above these lines. Can you quickly check if this makes sense to you?